### PR TITLE
Update R8 link in docs

### PR DESCRIPTION
**Pull request by yschimke**
Closes https://github.com/square/okhttp/issues/7044

### CODE REVIEW

File: docs/r8_proguard.md [hunk 1]
```diff
@@ -1,12 +1,13 @@
...
-If you use OkHttp as a dependency in an Android project which uses R8 as a default compiler you 
+If you use OkHttp as a dependency in an Android project which uses R8 as a default compiler you
 don't have to do anything. The specific rules are [already bundled][okhttp3_pro] into the JAR which can be
 interpreted by R8 automatically.
 
 If you, however, don't use R8 you have to apply the rules from [this file][okhttp3_pro]. You might
 also need rules from [Okio][okio] which is a dependency of this library.
 
- [okhttp3_pro]: https://github.com/square/okhttp/blob/master/okhttp/src/main/resources/META-INF/proguard/okhttp3.pro
+ [okhttp3_pro]: https://raw.githubusercontent.com/square/okhttp/master/okhttp/src/jvmMain/resources/META-INF/proguard/okhttp3.pro
```
Comment (yschimke) [inline docs/r8_proguard.md:12]: Changes to the raw file.

**Review by Goooler — COMMENTED** [review]
https://github.com/square/okhttp/blob/20d2fa4ffa6823687ff8dfdae7cbc82a6ad1d5f0/README.md?plain=1#L172

This line also need to be updated?

**Comment by Goooler** [comment]
https://github.com/square/okhttp/blob/20d2fa4ffa6823687ff8dfdae7cbc82a6ad1d5f0/README.md?plain=1#L177

And this line redirect to https://square.github.io/okhttp/features/r8_proguard/

**Comment by yschimke** [comment]
@Goooler thanks for the thorough review.

**Pull request merged**